manifests/l/LGUG2Z/whkd/0.2.1/LGUG2Z.whkd.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: LGUG2Z.whkd
+PackageVersion: 0.2.1
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+UpgradeBehavior: install
+Commands:
+- whkd
+ProductCode: '{A65D5F79-27CE-46E9-B3E1-04963AE29B28}'
+ReleaseDate: 2024-04-10
+AppsAndFeaturesEntries:
+- UpgradeCode: '{0E5D0A9F-3AEC-41B7-A580-095B9C29ABD6}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/LGUG2Z/whkd/releases/download/v0.2.1/whkd-0.2.1-x86_64.msi
+  InstallerSha256: B6A7C97E85BC94F1BDB65F55BEC9765F8A95B363905DF494A35B60F3B354FE03

```

Since you rebuilt the binaries in a new release, you need to update the hashes:
```suggestion
  InstallerSha256: C7A6EDD79F1A81A7CF081C9F50FF5277BDB83852618EFC6004AC9B3D631C6619
```